src/inc/corprof.idl

```diff
@@ -2375,6 +2379,29 @@ interface ICorProfilerCallback7 : ICorProfilerCallback6
 }
 
 
+[
+    object,
+    uuid(5BED9B15-C079-4D47-BFE2-215A140C07E0),
+    pointer_default(unique),
+    local
+]
+interface ICorProfilerCallback8 : ICorProfilerCallback7
+{
+    // This event is triggered whenever a metadata less method is jit compiled.
```

Comment:
update comment to say 'dynamic' instead of 'metadata less' now that the naming has been changed